src/pages/resultsView/mutation/MutationMapperStore.ts

```diff
@@ -120,4 +177,21 @@ export class MutationMapperStore {
     @cached get dataStore():IMobXApplicationDataStore<Mutation[]> {
         return new SimpleMobXApplicationDataStore<Mutation[]>(this.processedMutationData);
     }
+
+    @cached get pdbChainDataStore(): IMobXApplicationDataStore<IPdbChain> {
+        const dataStore = new SimpleMobXApplicationDataStore<IPdbChain>(this.mergedAlignmentData);
+
+        // define sort metric and direction at initialization time
+        dataStore.sortAscending = false;
+        dataStore.sortMetric = (pdbChain: IPdbChain) => [
+            pdbChain.identity,         // first, sort by identity
+            pdbChain.alignment.length, // then by alignment length
+            pdbChain.identityPerc,     // then by identity percentage
+            // current sort metric cannot handle mixed values so generating numerical values for strings
+            ...calcPdbIdNumericalValue(pdbChain.pdbId, true), // then by pdb id (A-Z): always returns an array of size 4
+            -1 * pdbChain.chain.charCodeAt(0)                 // then by chain id (A-Z): chain id is always one char
```

Comment:
I am not 100% satisfied with this solution...

@adamabeshouse, do we have a better way to solve the mixed type of sort metrics?